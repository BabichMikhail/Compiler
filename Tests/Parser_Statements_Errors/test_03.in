var a, b: integer; begin
	a b; 
end.