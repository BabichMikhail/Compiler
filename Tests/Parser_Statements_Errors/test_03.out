Error: Illegal Expression. Line 2 Column 7
